docs/examples/styled.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Styled
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This example shows how to provide styles to the image focal point component.
```

Comment:
This example shows how to override the standard component styles with your custom ones and apply them to the image focal point component.